src/job/models/newJobHandler.ts

```diff
@@ -50,7 +50,10 @@ export class NewJobHandler implements IJobHandler {
       await this.taskBuilder.pushTasks(job.id, mergeTasks);
 
       logger.info({ msg: 'Updating job with new metadata', ...metadata, extendedLayerMetadata });
-      await this.queueClient.jobManagerClient.updateJob(job.id, { parameters: { metadata: extendedLayerMetadata, partData, inputFiles } });
+      await this.queueClient.jobManagerClient.updateJob(job.id, {
```

Comment:
add test toHaveBeenCalledWith the keys of the update request

Reply:
added:
```
      expect(queueClientMock.jobManagerClient.updateJob).toHaveBeenCalledWith(job.id, {
        internalId: expect.any(String),
        parameters: expect.any(Object),
      });
```